backend/experiment/actions/final.py

```diff
@@ -36,6 +37,7 @@ class LogoConfiguration(TypedDict):
 class FinalActionResponse(TypedDict):
     view: str
     score: float
+    percentile: float
```

Comment:
I think `percentile` should be optional (it's not going to be applicable in every scenario and/or not every researcher might want to show it to the participants).